rs/replicated_state/src/canister_state/system_state/call_context_manager.rs

```diff
@@ -276,7 +276,7 @@ impl CallContextManagerStats {
 
     /// Calculates the stats for the given call contexts and callbacks.
     ///
-    /// Time complexity: `O(|call_contexts| + |callbacks|)`.
+    /// Time complexity: `O(n)`.
```

Comment:
Nit, but I found the previous formulation more informative, it's not really clear to me what `n` is if I just see that.

Reply:
I'd then rather write "EXPENSIVE - DO NOT CALL REPEATEDLY" or something along those lines. But I leave it up to you.